extensions/amp-carousel/0.2/amp-carousel.js

```diff
@@ -530,7 +530,10 @@ class AmpCarousel extends AMP.BaseElement {
 
       if (isSlides) {
         const wrapper = document.createElement('div');
-        wrapper.className = 'i-amphtml-carousel-slotted';
+        wrapper.classList.add([
```

Comment:
FMI, is there a benefit to use `classList.add()` over setting `className = 'i-amphtml-carousel-slotted i-amphtml-carousel-wrapper'`?

Reply:
Not that I know of. IMO it seemed cleaner.